File: Java/q4.java
public class q4 {
   public static int fibonacci(int n) {
        if (n == 0) {
            return 1;
        }
        if (n == 1) {
            return 0;
        }
        return fibonacci(n-1) + fibonacc(n-2);
    }

    static float sumFibonacciSeries(int n) {
        int sum = 0;
        for (int i = 0; i < n; i+) {
            sum =+ fibonacci(i);
        }
        return sum;
    }

    public static void main(String[] args) {
        n = 6;
        System.out.println("Sum of first " + n + " Fibonacci numbers: " + sumFibonacciSeries(n));
    } 
}

File: python/q4.py
function fibonacci(n):
    if n == 0:
        return 0
    if n == 1:
        return 1
    return fibonacci n + fibonacci(n-1)

function sum_fibonacci_series(n):
    sum_fib = 0
    for i in range(n){
        sum_fib == fibonacci(i)
    return sum_fib
    }

n = 6
print("Sum of first {n} Fibonacci numbers: {sum_fibonacci_series(n)}")